### PR TITLE
Updates on color palette

### PR DESCRIPTION
**Pull request by tarantilis**
https://taskman.eionet.europa.eu/issues/153269

Updates on link colors based on latest docusaurus update
https://62713c57bcbdf4004a3d99b8-jscqxuxtoe.chromatic.com/?path=/story/components-call-to-action--link

### CODE REVIEW

**Comment by tarantilis** [comment]
@eea-jenkins build storybook

File: theme/themes/eea/globals/site.variables [hunk 1]
```diff
@@ -194,9 +194,11 @@
        Links
 --------------------*/
 
-@linkColor           : @primaryColor;
+@linkColor           : @blue-4;
+@linkHoverColor      : @blue-5;
+@linkVisitedColor    : @blue-3;
```
Comment (ichim-david) [inline theme/themes/eea/globals/site.variables:199]: visited color is not accessible, I've added a comment in the ticket and we should get a new proposal from Manthos

**Review by ichim-david — CHANGES_REQUESTED** [review]

**Comment by ichim-david** [comment]
@g-stamatis  @tarantilis  see https://taskman.eionet.europa.eu/issues/153269#note-38

**Comment by ichim-david** [comment]
@g-stamatis check the ticket feedback, visited color has change and will need to be reflected in this pull request

**Comment by tarantilis** [comment]
@eea-jenkins build storybook

**Pull request merged**